new-site/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "hds-site",
+  "private": true,
+  "description": "Documentation for Helsinki Design System",
+  "version": "0.1.0",
+  "dependencies": {
+    "gatsby": "^3.13.1",
+    "gatsby-plugin-image": "^1.13.0",
+    "gatsby-plugin-manifest": "^3.13.0",
+    "gatsby-plugin-offline": "^4.13.0",
+    "gatsby-plugin-react-helmet": "^4.13.0",
+    "gatsby-plugin-sharp": "^3.13.0",
+    "gatsby-source-filesystem": "^3.13.0",
+    "gatsby-transformer-sharp": "^3.13.0",
+    "hds-core": "^1.5.0",
+    "hds-react": "^1.5.0",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-helmet": "^6.1.0"
+  },
+  "devDependencies": {
+    "hds-design-tokens": "^1.5.0",
```

Comment:
I wonder should this be in the dependencies section, as hds-core and hds-react are there too?

Reply:
- [ ] fix dependencies